src/mono/wasm/runtime/es6/dotnet.es6.pre.js

```diff
@@ -2,5 +2,7 @@ if (_nativeModuleLoaded) throw new Error("Native module already loaded");
 _nativeModuleLoaded = true;
 // see https://github.com/emscripten-core/emscripten/issues/19832
 Module["getMemory"] = function () { return wasmMemory; }
+Module["getWasmIndirectFunctionTable"] = function () { return wasmTable; }
 createDotnetRuntime = Module = createDotnetRuntime(Module);
 Module["getMemory"] = function () { return wasmMemory; }
+Module["getWasmIndirectFunctionTable"] = function () { return wasmTable; }
```

Comment:
I'm ok with having the `getMemory` and `getWasmIndirectFunctionTable` before `createDotnetRuntime` but I don't understand why it needs to also be after.

Reply:
https://github.com/dotnet/runtime/pull/92664#issuecomment-1738139068

Reply:
I don't understand that comment, could you please explain why it's necessary ?

Reply:
I still don't actually understand why it's broken without this, TBH